### ISSUE TITLE
Node Specs

### ISSUE THREAD

**Issue by daplcor** [kadena-io/chainweb-node#1827]
Please review the recommended specs for running nodes in the README. This comes up quite often in chat and private DMs.  I put this graphic together last week and hope it helps cover the general gist.  
![flow](https://github.com/kadena-io/chainweb-node/assets/95604731/dea71caa-863a-41c4-98e0-7d2e87c9abeb)

**Comment by chessai** [comment]
Hi @daplcor, are you referring to specifically the CPU/Memory requirements of a node? This graphic seems to imply something more; some node architecture.

**Comment by daplcor** [comment]
Yes, I believe it's just important to note the specs.  The graphic includes some suggested and doesn't need to be retained or used.
